styles/components/buttons.css

```diff
@@ -0,0 +1,56 @@
+@import url('../shared/common.css');
+@import url('../components/flexbox.css');
+@import url('../components/navbars.css');
+
+/********************************************** Buttons ******************************************/
+
+.btn {
+    outline: none;
+    display: flex;
+    align-items: center;
+    border-style: outset;
+    border-width: 0.125rem;
+    padding: 0.5rem 1rem;
+}
+
+.btn:active {
+    border-style: inset;
```

Comment:
was this inset intentional?

Reply:
yes shivam , for the click effect on buttons

Reply:
Oh! this is something new to me I defiantly need to read more about this. The rest of the stuff looks good I believe we are good to merege.